source/plugins/go/src/extension/extension.go

```diff
@@ -35,38 +36,33 @@ func GetInstance(flbLogger *log.Logger, containertype string) *Extension {
 	return singleton
 }
 
-func getExtensionConfigs() ([]ExtensionConfig, error) {
+func getExtensionData() (TaggedData, error) {
 	guid := uuid.New()
 	taggedData := map[string]interface{}{"Request": "AgentTaggedData", "RequestId": guid.String(), "Tag": "ContainerInsights", "Version": "1"}
 	jsonBytes, err := json.Marshal(taggedData)

```

Comment:
if there is any error marshalling taggedData to json, log the error message and return the error